components/org.wso2.carbon.data.provider/src/main/java/org/wso2/carbon/data/provider/internal/DashboardRestApiProvider.java

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
**Note**: Even though GitHub diff shows that this class is newly added, in fact it was moved to here from [`org.wso2.carbon.data.provider`](https://github.com/wso2/carbon-analytics/pull/1226/files#diff-7906381143fe441c1979e6bfafd4e307L19) package. See commit 8c78cfe83fec9180b34df5fa97a6b07c7e37fc95.